article/app/views/fragments/inArticleAudio.scala.html

```diff
@@ -0,0 +1,59 @@
+
+<div class="inline-audio_container">
+    <div class="inline-audio_icon">
+    @fragments.inlineSvg("football-weekly-300", "journalism/in-article-audio-player")
+    </div>
+    <div class="inline-audio_content">
+        <div class="inline-audio_content_header">
+            <h3>
+                <span class="kicker">Listen / </span>
+                Class, schoolings and some harsh lessons - Football Weekly
+            </h3>
+        </div>
+        <div class="inline-audio_content_player">
+            <audio
+            class="inline-audio-player-element"
+            src="https://flex.acast.com/audio.guim.co.uk/2018/08/20-54476-gnl.fw.20180820.sj.fw2008.mp3"
+            preload="metadata"
+            data-media-id="gu-audio-5b7467c1e4b016cd4306acc7"
+            data-title="Class, schoolings and some harsh lessons – Football Weekly"
+            data-component="in-article audio"
+            >
```

Comment:
On thurs: the `src`, `data-media-id`, `data-media-id`& `data-title` and default text should be changed to latest ep of Football Weekly